regression.py

```diff
@@ -193,6 +193,11 @@ def testGAMESS_GAMESS_US2013_N_UHF_out(logfile):
     """An UHF job that has an LZ value analysis between the alpha and beta orbitals."""
     assert len(logfile.data.moenergies) == 2
 
+def testGAMESS_GAMESS_US2014_CdtetraM1B3LYP_log(logfile):
+    """This logfile had coefficients for only 80 molecular orbitals."""
+    assert len(logfile.data.mocoeffs) == 2
+    assert numpy.count_nonzero(logfile.data.mocoeffs[0][80:,80:]) == 0

```

Comment:
Sorry it took so long to respond. It was to help me remember that there's data for the 79th MO but not the 80th, because I was confused trying to understand the test and the logfile, and the zero-based indexing kept throwing me off. Also, I think it's data for 79 MOs, not 80. That might have confused me too.